pkg/process/metadata/parser/service.go

```diff
@@ -334,3 +344,17 @@ func parseCommandContextJava(args []string) string {
 
 	return "java"
 }
+
+func advancedGuessJavaServiceName(se *ServiceExtractor, process *procutil.Process, args []string, jarname string) (string, bool) {
+	if se.useImprovedAlgorithm {
+		// try to introspect the jar to get service name from spring application name
+		// TODO: pass process envs
+		springAppName, err := javaparser.GetSpringBootAppName(process.Cwd, jarname, args)
+		if err != nil {
+			log.Tracef("Error while trying to extract properties from potential spring boot application: %v", err)
```

Comment:
Is this kind of verbosity acceptable if the user switch to log level trace? Because there are also other logs that at this point should be removed. Otherwise we can keep those logs in order to troubleshoot if needed

Reply:
So, most of the customers do not use Trace level on their own good will, but only during investigation

assuming you have 100 logs like that, would it give you any clue what went wrong and why?

if you need more information - then let's add it, if the error will be logged 100 times or 10 will it make the difference?